x-pack/plugins/spaces/server/lib/get_space_selector_url.ts

```diff
@@ -0,0 +1,9 @@
+/*
```

Comment:
__note:__ looks like we still have `x-pack/legacy/plugins/spaces/server/lib/get_space_selector_url.ts`, should we delete it?

Reply:
Yes good catch. I'll remove.